src/stepDefinitions/filesContext.js

```diff
@@ -270,3 +269,22 @@ Given(/^user "([^"]*)" has created folder "(.+)"$/, function(userId, folderName)
   folderName = folderName.replaceAll("\\", "")
   return webdavHelper.createFolder(userId, folderName)
 })
+
+Then(
+  'as user {string} folder {string} should contain {string} items', async function (
+    userName,
+    folderName,
+    itemsCount
+) {
+  const davPath = webdavHelper.createDavPath(userName, folderName)
+  const resourcesInsideFolder = await webdavHelper.propfind(davPath, userName, [])
+  const jsonResponse = convert.xml2js(resourcesInsideFolder)
+  let actualCount = jsonResponse.elements[0].elements.length
```

Comment:
`jsonResponse.elements[0].elements.length` does not give much of the information to the reader. Can you add a comment on this line as well? Which element that you are targeting?